### DISCUSSION TITLE
Is it safe?

### DISCUSSION THREAD

**Discussion by HarsyCZ** [SpotX-Official/SpotX#499] [Question] [answered]
Hello, ive been using spotx for 1,5year now, but with the latest update the instalation progress changed and im not sure if allowing it to my powershell is safe. If i try to go to a page where it will download from using the powershell my antivirus completly blocks the site, any answers pls?

![x](https://github.com/amd64fox/SpotX/assets/146195321/e3b6c331-7e6b-4821-8b6a-8f667641c54d)

**Comment by amd64fox** [comment]
> instalation progress changed

what exactly ?

**Reply by HarsyCZ** [reply]
allowing the security protocol thing on powershell

**Reply by amd64fox** [reply]
If you are talking about this:
```ps1
[Net.ServicePointManager]::SecurityProtocol = 3072
```
then this command does the same thing as this one:
```ps1
[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```
what does your AV have to do with it and why is it blocking some website?

I think you should read the FAQ before formulating your question

**Comment by Staffed** [comment, marked as answer]
Yes it is safe, from my understanding is that the bat files change the spotify files which pisses off the antivirues.

**Comment by gmy77** [comment]
Safe, verified and about my personal view... 100% SAFE. Then... i have to say thanks to DEVS... works perfectly.

**Comment by prexol21** [comment]
for some reason my antivirus didnt say anything about that file just my windows protection because this is a file that changes spotfiy orig files and pretend to be a spotfiy premium thingy but cant play along with other people on discord for some reason
